### ISSUE TITLE
Vendor recipe respawn times

### ISSUE THREAD

**Issue by Scoopdunker** [dalaranwow/dalaran-wow#4640]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
waited at vendor to buy the recipe for 2 hours last night, logged out at vendor and logged in a few times and still no respawn. I've seen this complaint is pretty common for a vairety of recipe respawn times.
**Expected behaviour**: Tell us what should happen instead
II would think vendor recipes would respawn at least every 30 mins.
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TheRealestMcCoy** [comment]
Links to what recipes you are talking about would be a good start.
And proof of respawn timers also helps.

**Comment by Rushor** [comment]
no links so we cannot handle this issue.

the current list with all creatureids that sold items with 2 hours (7200seconds) restoretime:

```sql
(creatureid=777, 0, 5772, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=844, 0, 785, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=844, 0, 2449, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=844, 0, 2453, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=844, 0, 3355, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=844, 0, 3356, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=1250, 0, 6272, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=1307, 0, 929, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=1313, 0, 785, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=1313, 0, 2449, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=1313, 0, 2453, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=1313, 0, 3355, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=1313, 0, 3356, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=1318, 0, 10938, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=1318, 0, 10940, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=1453, 0, 929, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=1454, 0, 5772, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=1457, 0, 929, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=1474, 0, 5772, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=2140, 0, 858, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=2393, 0, 6054, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=2393, 0, 20971, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=2394, 0, 6274, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=2394, 0, 6401, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=2481, 0, 929, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=2481, 0, 6068, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=2663, 0, 10728, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=2685, 0, 13310, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=2687, 0, 13311, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=2688, 0, 4389, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=2688, 0, 10609, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=2805, 0, 929, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=2821, 0, 21942, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=2840, 0, 4778, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3005, 0, 5771, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3005, 0, 5772, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3005, 0, 10311, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3005, 0, 10325, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3012, 0, 6349, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3012, 0, 6377, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3012, 0, 10938, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=3012, 0, 10940, 4, restoretime_in_seconds_7200, 0, 0);
(creatureid=3019, 0, 12249, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3029, 0, 6365, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3096, 0, 4790, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3096, 0, 4793, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3096, 0, 4792, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3134, 0, 929, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=3180, 0, 4378, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=3333, 0, 6533, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=3335, 0, 5640, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=3335, 0, 5643, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3346, 0, 6346, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3346, 0, 6349, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3346, 0, 10938, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=3346, 0, 10940, 4, restoretime_in_seconds_7200, 0, 0);
(creatureid=3348, 0, 13478, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3356, 0, 12162, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3364, 0, 5772, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3364, 0, 6270, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3364, 0, 6274, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3364, 0, 10314, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3364, 0, 10317, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3366, 0, 18731, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3367, 0, 20856, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3485, 0, 5772, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3485, 0, 6272, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3490, 0, 765, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=3490, 0, 785, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=3490, 0, 2447, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=3490, 0, 2449, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3490, 0, 2453, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3490, 0, 6053, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3499, 0, 5640, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3499, 0, 6272, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3499, 0, 20855, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3534, 0, 929, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=3537, 0, 5772, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3556, 0, 5771, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3556, 0, 5786, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3556, 0, 5787, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3556, 0, 6272, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3572, 0, 6533, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3683, 0, 5772, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3956, 0, 929, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=4083, 0, 2449, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4083, 0, 2453, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=4083, 0, 3356, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4083, 0, 3385, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=4083, 0, 6055, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4168, 0, 6272, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4173, 0, 11303, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=4173, 0, 11306, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4173, 0, 11308, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4175, 0, 12256, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4189, 0, 5772, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=4225, 0, 7451, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4229, 0, 11223, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4232, 0, 12249, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4235, 0, 12247, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4561, 0, 16217, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4561, 0, 20854, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4577, 0, 5772, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4617, 0, 10938, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4617, 0, 10940, 4, restoretime_in_seconds_7200, 0, 0);
(creatureid=4775, 0, 20975, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4877, 0, 5973, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4877, 0, 20970, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4878, 0, 785, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=4878, 0, 929, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=4878, 0, 1711, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=4878, 0, 1712, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4878, 0, 2290, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=4878, 0, 3012, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4878, 0, 3356, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4878, 0, 3357, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=5158, 0, 10938, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=5158, 0, 10940, 4, restoretime_in_seconds_7200, 0, 0);
(creatureid=5411, 0, 6047, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=5594, 0, 6057, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=5594, 0, 9305, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=5757, 0, 6346, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=5757, 0, 10938, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=5757, 0, 10940, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=5758, 0, 10938, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=5758, 0, 10940, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=5940, 0, 6365, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=5940, 0, 6533, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=5944, 0, 5772, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=6567, 0, 4355, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=6568, 0, 7088, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=7854, 0, 7451, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=7854, 0, 8409, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=8158, 0, 3928, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=8158, 0, 6057, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=8158, 0, 6149, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=8159, 0, 12253, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=8305, 0, 785, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=8305, 0, 2449, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=8305, 0, 2453, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=8363, 0, 21948, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=8678, 0, 10607, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=8679, 0, 10602, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=8878, 0, 10858, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=9551, 0, 11304, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=9551, 0, 11305, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=9636, 0, 7114, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=9636, 0, 20973, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=11188, 0, 3928, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=11188, 0, 6149, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=11189, 0, 21957, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=11874, 0, 12253, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=11874, 0, 15726, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12022, 0, 16224, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12031, 0, 6365, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12031, 0, 6533, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=12043, 0, 11039, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12043, 0, 11101, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12245, 0, 1710, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=12245, 0, 3735, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12245, 0, 3827, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=12245, 0, 5489, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12245, 0, 12228, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12245, 0, 12231, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12245, 0, 12232, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12245, 0, 12240, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=3413, 0, 22729, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12941, 0, 21954, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12943, 0, 15741, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12956, 0, 15724, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12956, 0, 15762, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12957, 0, 15729, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12957, 0, 15751, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12962, 0, 6533, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=13476, 0, 929, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14740, 0, 6533, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 785, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 2449, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 3355, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 3357, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 3356, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 2453, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 27501, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 954, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 3012, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 4422, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 3928, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 3385, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 1181, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 858, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 4421, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 1477, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 2290, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=15179, 0, 21952, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=15419, 0, 10938, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=15419, 0, 10940, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=16015, 0, 8846, 5, restoretime_in_seconds_7200, 0, 0);
(creatureid=16015, 0, 14256, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=16224, 0, 5771, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=16588, 0, 22900, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=16635, 0, 10938, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=16635, 0, 10940, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=16641, 0, 785, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=16641, 0, 2449, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=16641, 0, 2453, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=16641, 0, 3355, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=16641, 0, 3356, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=16641, 0, 22900, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=16705, 0, 785, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=16705, 0, 2449, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=16705, 0, 2453, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=16705, 0, 3355, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=16705, 0, 3356, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=16705, 0, 22900, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=16722, 0, 10938, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=16722, 0, 10940, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=18011, 0, 21898, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=18011, 0, 21899, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=18753, 0, 10938, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=18753, 0, 10940, 4, restoretime_in_seconds_7200, 0, 0);
(creatureid=18773, 0, 10938, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=18773, 0, 10940, 3, restoretime_in_seconds_7200, 0, 0);
(creatureid=18802, 0, 22900, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=18951, 0, 10938, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=18951, 0, 10940, 4, restoretime_in_seconds_7200, 0, 0);
(creatureid=19042, 0, 22901, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=19053, 0, 30745, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=19053, 0, 30746, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=19053, 0, 30747, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=19053, 0, 30748, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=19234, 0, 10938, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=19234, 0, 10940, 4, restoretime_in_seconds_7200, 0, 0);
(creatureid=19537, 0, 10938, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=19537, 0, 10940, 4, restoretime_in_seconds_7200, 0, 0);
(creatureid=19663, 0, 10938, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=19663, 0, 10940, 4, restoretime_in_seconds_7200, 0, 0);
(creatureid=19836, 0, 23807, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=19836, 0, 23803, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=19042, 0, 22911, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 4425, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 10309, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 27498, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 4419, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 10308, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 27499, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 10305, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 27500, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 10306, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 10307, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 27502, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 4426, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 10310, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14847, 0, 27503, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 22789, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 22787, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 23793, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 22790, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 22791, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 22792, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 22793, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=14846, 0, 22794, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=2482, 0, 12163, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=15165, 0, 37915, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=12943, 0, 15725, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=4878, 0, 6068, 1, restoretime_in_seconds_7200, 0, 0);
(creatureid=33597, 0, 10940, 4, restoretime_in_seconds_7200, 0, 0);
(creatureid=33597, 0, 10938, 2, restoretime_in_seconds_7200, 0, 0);
(creatureid=33676, 0, 10940, 4, restoretime_in_seconds_7200, 0, 0);
(creatureid=33676, 0, 10938, 1, restoretime_in_seconds_7200, 0, 0);
```
